ethical-advertising-update-2019.rst

```diff
@@ -0,0 +1,117 @@
+.. post:: May 28, 2018
+   :tags: advertising, business, sustainability, ethical ads
+   :author: David
+   :location: SAN
+
+Ethical Advertising Update
```

Comment:
Needs a better title. I think we're really doing two things here, which maybe should be split into two posts:

* An ethical ads update
* Announcing a new ad format and expanding where we show ads

It definitely feels like burying the lede to do the later in a blog post called "ad updates"

Reply:
Agreed 100% on the title.

I completely understand this concern although I do want to build value for what we're doing before I tell people more ads are coming.

I see a few options here:

- The title could be changed to something like "Ethical Advertising Update & Coming Changes" to make sure to note that changes are coming.
- We could break the blog into two although then we are going to have a much shorter "build value" section before we talk about more ads. I also feel that the [blog for Alabaster ads](https://blog.readthedocs.com/ads-on-other-themes/) was a little awkward in that it just informed people ads were coming without doing enough to explain how users benefit from additional support, etc. Or rather, it reversed what I think of as the logical order of those things.
- We could refactor the blog a bit to mention some of the pending changes earlier and go into more details further down.

Thoughts?

Reply:
I think we need to at least mention a new ad format in the title and in the introduction. I don't think it's awkward to tell people ads are coming, when we want to communicate to them ads are coming :) If we're going to email people this post who are going to get ads on their docs to explain this to them, the current structure is not ideal.

Reply:
Sounds good. I'll change this post to just be an ethical ads update. I'll create another post for the changes.